src/components/ServiceList.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import styled from 'styled-components'
+import PlaneIcon from '../icons/PlaneIcon'
+import HotelIcon from '../icons/HotelIcon'
+import TrainIcon from '../icons/TrainIcon'
+import CarIcon from '../icons/CarIcon'
+
+const List = styled.ul`
+  padding-left: 0;
+  list-style-type: none;
+  display: flex;
+  > * {
+    background-color: #147cba;
+    color: white;
+    padding: 0.2em 0.4em;
+    border-radius: 3px;
+    text-transform: uppercase;
+    margin-right: 1em;
+  }
+`
+
+type Props = {
+  services: Array<string>
+}
+
+const serviceIcon = (service: string) => {
+  if (service === 'flights')
+    return (
+      <li key={service}>
```

Comment:
Can you add a new `styled` component for the `li` elements, with `display: flex` and `align-items: center`? I think it will look better.

You can also add a space between the icon and the name of the service to make it more readable.